### PR TITLE
Move search box into the right box

### PR DESCRIPTION
**Pull request by curran**
image

### CODE REVIEW

**Comment by curran** [comment]
image

File: index.html [hunk 1]
```diff
@@ -42,7 +42,7 @@
                 <h4></h4>
               </div>
               <div>
-                <input id="josquin_catalog" name="josquin_catalog" type="text" list="catalog">
+                <input title="Delete the text to see an autocomplete list." id="josquin_catalog" name="josquin_catalog" type="text" list="catalog">
```
Comment (seemantk) [inline index.html]: nice!

File: js/dev/baton.js [hunk 2]
```diff
@@ -180,7 +180,7 @@ function chartify(data) {
 
 
     // Titles and polish
-    var titles = divMeta.select(".panel-heading").selectAll("*")
+    var titles = divMeta.select(".meta-titles").selectAll("*")
```
Comment (seemantk) [inline js/dev/baton.js:183]: great to disambiguate this finally.

Comment (curran) [reply js/dev/baton.js:183]: Indeed. The original was causing the search box elements to get deleted somehow.

**Pull request merged**